tiny_dnn/core/framework/tensor.h

```diff
@@ -312,16 +297,50 @@ auto host_data() {
     return *this;
   }
 
+  /**
+   * Fill tensor with another tensor repeated size times over first axis
+   * @tparam S storage of parameter tensor
+   * @param size
+   * @param tensor
+   * @return
+   */
+  template <typename S>
+  Tensor &fill(size_t size, Tensor<U, S> tensor) {
```

Comment:
we should rename this to `Tensor::repeat(...)` it wasn't clear to me until I read the comments above